www/source/partials/docs/_dev-pkgs-write-plans.html.md.erb

```diff
@@ -27,6 +27,8 @@ pkg_bin_dirs=(bin)
 
 It has the name of the software, the version, where to download it, a checksum to verify the contents are what we expect, run dependencies on `core/glibc` and `core/readline`, build dependencies on `core/coreutils`, `core/make`, `core/gcc`, libraries files in `lib`, header files in `include`, and a binary file in `bin`. Also, because it's a core plan, it has a description and upstream URL for the source project included.
 
+> Note: the `core` prefix is the origin of those dependencies. For more information, see [Create an Origin](/tutorials/download/create-origin/)
```

Comment:
Looks like there's a typo in the beginning of this note. "Note: The `core`...."

Reply:
@davidwrede fixed :)